### ISSUE TITLE
Creative menu out of order

### ISSUE THREAD

**Issue by RogueLogix** [RogueLogix/BiggerSeries#10]
_im not sure how to fix this_

**Comment by RogueLogix** [comment]
so, it appears to be based on the order that items are added to the item group, so, now i need to figure out how to specify that to my registry

**Comment by XFactHD** [comment]
Take a look [here](https://github.com/XFactHD/AdvancedTechnology/blob/1.15.2/src/main/java/xfacthd/advtech/common/data/ItemGroups.java), that's how I do it with a Comparator

**Comment by gizmocodes** [comment]
Thanks @XFactHD, based off that I got it to be sorted alphabetically for right now (31f17ac49ea107b488b5d090b38c091dbcf5a1a7; wasn't too bad actually). Later we'll dig in and properly sort (grouping by item type and what not), but this works for now (and is consistent, so items don't shuffle around every time you load the game). Possibly either by Forge tags or an "item weight" system? We'll see when we get to it.

Closing for now, can reopen or update later when we actually do it.
